src/baldr/graphtile.cc

```diff
@@ -8,6 +8,7 @@
 #include "midgard/logging.h"
 #include "midgard/pointll.h"
 #include "midgard/tiles.h"
+#include "worker.h"
```

Comment:
I'd actually like to take `valhalla_exception_t` out of `worker.h`. lots of files have this included only bcs of the exception. and that carries big headers itself.

Reply:
done in #5572